packages/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Any thoughts about naming our end to end tests package something like `e2e` instead of the technology we use (`cypress`)

Reply:
would still have to name a folder `cypress` in order to use the framework but i put it into a e2eTesting folder

Reply:
oo I wasn't aware of that, nvm mind my comment then!